File: app/src/main/java/com/wj/gradle/plugin/MainActivity.java
package com.wj.gradle.plugin;

import android.app.Activity;
import android.os.Bundle;
import android.text.Html;
import android.util.Log;
import android.widget.TextView;

import com.wj.gradle.plugin.annotation.WjRouter;

import androidx.annotation.Nullable;

@WjRouter(key = "/main.html?key=999")
public class MainActivity extends Activity {
    private TextView tvLongText;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Log.d("", getApplication().toString());
        String info = null;
        //LogUtils.logV("" + info.toString());
        tvLongText= findViewById(R.id.tv_long_text);
        tvLongText.setText("11");
        Html.fromHtml("11");
    }
}

File: app/src/main/java/com/wj/gradle/plugin/MainActivity.kt
package com.wj.gradle.plugin

import android.app.Activity
import android.os.Bundle
import android.widget.ImageView
import com.bumptech.glide.Glide
import com.bumptech.glide.load.engine.DiskCacheStrategy

class MainActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val code = ByteCode()
        code.sumMethod(1, 5)
        // code.stringMethod()
        val imageView = findViewById<ImageView>(R.id.iv_glide)

        Glide.with(this).load("http://...(url address)")
            .skipMemoryCache(true)
            .diskCacheStrategy(DiskCacheStrategy.ALL)
            .
            .into(imageView)
    }
}